src/OrderBundle/Block/RecentOrdersBlockService.php

```diff
@@ -73,7 +76,7 @@ public function execute(BlockContextInterface $blockContext, Response $response
         $criteria = [];
 
         if ('admin' !== $blockContext->getSetting('mode')) {
-            $orders = $this->orderManager->findForUser($this->securityContext->getToken()->getUser(), ['createdAt' => 'DESC'], $blockContext->getSetting('number'));
+            $orders = $this->orderManager->findForUser($this->tokenStorage->getToken()->getUser(), ['createdAt' => 'DESC'], $blockContext->getSetting('number'));
```

Comment:
Could you break this line, we follow limit of 120 characters

Reply:
I though about it, but running `php-cs-fixer` didn't change that....

Will fix now